pkgs/tools/misc/idutils/default.nix

```diff
@@ -1,27 +1,31 @@
-{ fetchurl, lib, stdenv, emacs }:
+{ fetchurl, lib, stdenv, emacs, gnulib, autoconf, bison, automake, gettext, gperf, texinfo, perl, rsync, gawk }:
 
 stdenv.mkDerivation rec {
-  name = "idutils-4.6";
+  pname = "idutils";
+  version = "4.6";
 
   src = fetchurl {
-    url = "mirror://gnu/idutils/${name}.tar.xz";
+    url = "mirror://gnu/idutils/${pname}-${version}.tar.xz";
```

Comment:
```suggestion
    url = "mirror://gnu/idutils/idutils-${version}.tar.xz";
```

Reply:
@SuperSandro2000  We were inserting these silly pname's for years and now you want to revert that one at a time.

Reply:
Fixed